.github/config/distribution_matrix.json

```diff
@@ -39,5 +39,21 @@
         "vcpkg_triplet": "x64-windows-static-md"
       }
     ]
+  },
+  "wasm": {
```

Comment:
Is there a reason there are 3 architectures here, but only 2 in the backport in [_extension_distribution.yml](https://github.com/duckdb/duckdb/pull/9356/files#diff-feabb7d2bbf4ed78e4061049b5a95bf92059972e8e132e391902b6cc148bff13) ?

Reply:
Nicely spotted, I went back and forth on this, but probably it's simpler and better to have them uniform.